playbooks/06-deploy-va.yml

```diff
@@ -0,0 +1,38 @@
+- name: Run pre_deploy hooks
+  vars:
+    hooks: "{{ pre_deploy | default([]) }}"
+    step: pre_deploy
+  ansible.builtin.import_playbook: ./hooks.yml
+
+- name: Deploy podified control plane
+  hosts: "{{ cifmw_target_host | default('localhost') }}"
+  gather_facts: false
+  tasks:
+    - name: Load parameters files
+      ansible.builtin.include_vars:
+        dir: "{{ cifmw_basedir }}/artifacts/parameters"
+
+    - name: Configure Storage Class
+      ansible.builtin.include_role:
+        name: ci_local_storage
+
+    - name: Check requirements
```

Comment:
why not rely on the `main.yml` ? It does the same things in the same order afaik?

Reply:
(ok, HCI needs ctlplane-1 -> dataplane-1 -> ceph deploy -> ctlplane-2 -> dataplane-2)